### PR TITLE
First draft of creating images chapter

### PR DESCRIPTION
**Pull request by OliverWoolland**

### CODE REVIEW

File: code/Dockerfile
Comment (fherreazcue) [inline code/Dockerfile:1]: why not put this in another directory? as in SPUCv2 or something?

File: code/SPUC/spuc.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 # ------------------------------------------------------------------------------
 # Endpoint for exporting the unicorn sightings if the EXPORT environment variable is set to True
-if os.environ.get("EXPORT").lower() == "true":
+if os.environ.get("EXPORT") == "True":
```
Comment (fherreazcue) [inline code/SPUC/spuc.py:37]: Is this more restricting intentionally, or was it not working?

**Review by fherreazcue — CHANGES_REQUESTED** [review]
Text I will review on my pass, probably better to look at it on the site, so after merge...

Other things just take as questions...

File: code/SPUC/spuc.py (hunk 1 above)
Comment (OliverWoolland) [reply code/SPUC/spuc.py:37]: Not working :'(

**Comment by fherreazcue** [comment]
Still need to review text, but merging to make our lives easier...

**Pull request merged**